### ISSUE TITLE
Loading zenodo SSCx anatomy model, snap is not able to handle the undefined "biophysical_neuron_models_dir" gracefully

### ISSUE THREAD

**Issue by markovg** [BlueBrain/snap#197]
Hi,
I'm trying to use bluepysnap with this dataset:
https://zenodo.org/record/6906785

I'm getting this issue:
![image](https://user-images.githubusercontent.com/427643/235582710-a1711c51-4dfb-4ffa-8cfc-2d960dd6fefa.png)

**Comment by mgeplf** [comment]
Hey stranger,

Unfortunately we've had to extend some of the configs in SONATA to handle extra complexity we've run into.  Getting these upstream has not been successful, so we've started our documentation here:
https://sonata-extension.readthedocs.io/en/latest/
and https://sonata-extension.readthedocs.io/en/latest/sonata_config.html for circuit_config.json files.

Since it's an anatomy model, one could add:
```
"metadata": { "status": "partial" }
```
At the top of the `circuit_config.json`, and as long as one is using `libsonata==0.1.18` it should work.
